include/micm/configure/solver_config.hpp

```diff
@@ -21,52 +22,98 @@
 #include <micm/system/species.hpp>
 #include <micm/system/system.hpp>
 #include <micm/util/constants.hpp>
+#include <micm/util/error.hpp>
 #include <nlohmann/json.hpp>
 #include <sstream>
 
-namespace micm
+enum class MicmConfigErrc
+{
+  Success = MICM_CONFIGURATION_ERROR_CODE_SUCCESS,
```

Comment:
Now that we have a better error system, should we think about removing the success status since it's not an error? Rather than checking the return value throughout the configuration, we can throw an error when we arrive at one

Reply:
Good point. I'll do that now.

Reply:
done!